### DISCUSSION TITLE
Setting domain at Organization level

### DISCUSSION THREAD

**Discussion by pdifeo** [postalserver/postal#2738] [Help with using Postal]
Can someone explain to me what is the point of configuring domains at organization level?

<img width="324" alt="image" src="https://github.com/postalserver/postal/assets/8618744/3d45c6cc-5774-4d15-9934-7dce2de0032c">

**Comment by Pacerino** [comment]
We use this to separate our services. This allows us to manage the webhooks and routes per mail server. For example, Mautic has its own server and our internal application has its own. So we can send a webhook for bounces only to Mautic. The domain at the level of the organisation allows us to easily set up and manage the domain. Also we can setup a dev server to develop on this server with every domain we have.

**Reply by willpower232** [reply]
This is a great description, I would just add that a domain at the organisation level applies to all servers within the organisation which enables any combination of sharing the domain name for whatever reason you have.

**Reply by hvisage** [reply]
What confuses me, is that you can have the same domain on both organizational and server level...
